Makefile

```diff
@@ -5,7 +5,7 @@ LDFLAGS += -X "github.com/pingcap/dm/pkg/utils.GitBranch=$(shell git rev-parse -
 LDFLAGS += -X "github.com/pingcap/dm/pkg/utils.GoVersion=$(shell go version)"
 
 CURDIR   := $(shell pwd)
-GO       := GO111MODULE=on go
+GO       := GO111MODULE=on GOOS=linux GOARCH=amd64 go
```

Comment:
i think we should't add `GOOS=linux GOARCH=amd64` here, because many developer also use mac and windows

Reply:
yes，because build binary testing, so i modify